src/Tag/Base/NormalTag.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Html\Tag\Base;
+
+use Yiisoft\Html\Html;
+
+abstract class NormalTag extends Tag
+{
+    private bool $encode = true;
+    private bool $doubleEncode = true;
+    private string $content = '';
+
+    /**
+     * @return static
+     */
+    final public function withoutEncode(): self
```

Comment:
```suggestion
    final public function encode(bool $encode): self
```

It could be useful for:

```php
MyNormalTag::tag()->encode($this->encode);
```

Reply:
All methods withoutEncode() and similar refactored to encode(bool $encode).